applications/asset_tracker_v2/doc/debug_module.rst

```diff
@@ -38,6 +38,19 @@ This enables the application to be able to collect coredump data before a reboot
 To enable Memfault, you must include the :file:`../overlay-memfault.conf` when building the application.
 To get started with Memfault, see :ref:`using_memfault`.
 
+.. _asset_tracker_v2_ext_transport:
+
+Custom transport
+----------------
+
+The data that is collected from the device can be routed through an custom transport to the Memfault cloud instead of using Memfault's own HTTPS.
```

Comment:
```suggestion
The data that is collected from the device can be routed through a custom transport to the Memfault cloud instead of using Memfault's own HTTPS transport.
```